modules/shared/src/main/scala/org/tessellation/schema/package.scala

```diff
@@ -54,15 +54,33 @@ trait OrphanInstances {
   implicit val posBigIntShow: Show[PosBigInt] =
```

Comment:
Do we need the show/decoder/encoder for Pos/NonNeg BigInts still?

Reply:
I don't think so. Removed :+1: